### ISSUE TITLE
macros fail to compile with latest Rust nightly build

### ISSUE THREAD

**Issue by fuchsnj** [tomjakubowski/json_macros#18]
```
#![feature(plugin)]
#![plugin(json_macros)]

fn main() {
	let json = json!({
		"test" : 4		
	});
}
```

```
src\main.rs:5:13: 7:4 error: expected identifier, found `_ob`
src\main.rs:5 	let json = json!({
src\main.rs:6 		"test" : 4		
src\main.rs:7 	});
Could not compile `json_test`.

To learn more, run the command again with --verbose.
```

```
rustc --version
rustc 1.1.0-nightly (c4b23aec4 2015-04-29) (built 2015-04-29)
```

**Comment by tomjakubowski** [comment]
Seems related to this: https://github.com/rust-lang/rust/pull/23085

Fix incoming.

**Comment by tomjakubowski** [comment]
Released in v0.2.4
